changelog.txt

```diff
@@ -1,7 +1,7 @@
 Version 2.1.1
 ----------
 - Introduces result sharing to minimize duplicate_command errors
-- Device Code authorization endpoint is now sourced from .Well-Known config
+- No longer query well known config to obtain token endpoint - build it manually instead
```

Comment:
I mean, if this is a revert, then nothing changes, right?

Reply:
Nothing changes for Device Code flow, hence I removed that line. But I added a new line about reverting using well known config for Token Endpoint as that change was made back in November 2019 here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/714